src/modules/userMethods.ts

```diff
@@ -339,6 +340,38 @@ export const userMethods = {
         });
     },
 
+    editUserFields: async (data: EditUserFields, user: User) => {
+        const fieldsToUpdate: Record<string, string> = {};
+
+        if (data.organizationUnitId && data.organizationUnitId !== user.organizationUnitId) {
+            const newOrganization = await prisma.organizationUnit.findUnique({
+                where: {
+                    id: data.organizationUnitId,
+                },
+            });
+
+            if (newOrganization) {
+                fieldsToUpdate.organizationUnitId = newOrganization.id;
+            }
+        }
+
+        if (data.name && data.name !== user.name) {
+            fieldsToUpdate.name = data.name;
+        }
+        if (data.supervisorId && data.supervisorId !== user.supervisorId) {
```

Comment:
do we really need those checks for unchanged fields? they blow up the line count so much